config/wireshark.mpb

```diff
@@ -35,37 +32,41 @@ project : glib {
   verbatim(nmake, top, 1) {
     include $(WIRESHARK_SRC)/config.nmake
   }
-
-  specific(!prop:windows) {
-    libpaths    += $(WIRESHARK_SRC)/wiretap/.libs $(WIRESHARK_SRC)/epan/.libs
-    lit_libs    += wiretap wireshark
-  }
 }
 
+// Autoconf / In Source Wireshark Build Method
+// This one is how the Wireshark Developers Guide says to build Wireshark on
+// Linux (autogen.sh).
+// This is also used for older 1.x Wireshark Windows builds.
 feature(wireshark) {
-  avoids += wireshark2
+  avoids += wireshark_cmake
   specific(prop:windows) {
     libpaths    += $(WIRESHARK_SRC)/wiretap $(WIRESHARK_SRC)/epan
     lit_libs    += wiretap-$(WIRETAP_VERSION) libwireshark
+  } else {
+    libpaths    += $(WIRESHARK_SRC)/wiretap/.libs $(WIRESHARK_SRC)/epan/.libs
+    lit_libs    += wiretap wireshark
   }
 }
 
-feature(wireshark2) {
+// CMake out of source builds of Wireshark are used in the Developers Guide
+// for Windows and macOS. This can also be used on Linux if built using CMake.
+// To use build against them we need two absolute
+// paths:
+//   WIRESHARK_BUILD
+//     The location of the build that the user choose when building Wireshark.
+//     It must contain the config.h file after building Wireshark.
+//   WIRESHARK_LIB
```

Comment:
Since these libraries are always under WIRESHARK_BUILD, should we make it so that WIRESHARK_LIB is just the relative path?